packages/commonwealth/client/scripts/models/Reaction.ts

```diff
@@ -1,6 +1,29 @@
 export type ReactionType = 'like';
 import type moment from 'moment';
-import { addressToUserProfile, UserProfile } from './MinimumProfile';
+
+export type UserProfile = {
+  id: number;
+  name: string;
+  address: string;
+  lastActive: string;
+  avatarUrl: string;
+};
+
+export function addressToUserProfile(address): UserProfile {
+  const profile = address?.User?.Profiles[0];
+  if (!profile) {
+    // @ts-expect-error <StrictNullChecks/>
+    return undefined;
+  }
+
+  return {
+    id: profile.id,
+    avatarUrl: profile?.avatar_url,
+    name: profile?.profile_name || profile?.name,
+    address: address?.address,
+    lastActive: address?.last_active,
+  };
+}
```

Comment:
lets not duplicate this

Reply:
This needs to be duplicated because before it was being imported from the server. I need to completely separate the server from the client in order to deploy them separately in a followup PR.

Reply:
this was previously imported from `MinimumProfile.tsx` which is in frontend. I still think this change can be removed